apis/Google.Cloud.Speech.V1Beta1/Google.Cloud.Speech.V1Beta1/SpeechRecognizer.cs

```diff
@@ -0,0 +1,98 @@
+        }
+
+        public SyncRecognizeResponse Recognize(RecognitionConfig config, RecognitionAudio audio, CallSettings callSettings = null)
+            => _client.SyncRecognize(
+                GaxPreconditions.CheckNotNull(config, nameof(config)),
+                GaxPreconditions.CheckNotNull(audio, nameof(audio)),
+                callSettings);
+
+        public Task<SyncRecognizeResponse> RecognizeAsync(RecognitionConfig config, RecognitionAudio audio, CallSettings callSettings = null)
+            => _client.SyncRecognizeAsync(
+                GaxPreconditions.CheckNotNull(config, nameof(config)),
+                GaxPreconditions.CheckNotNull(audio, nameof(audio)),
+                callSettings);
+
+        public Task<SyncRecognizeResponse> RecognizeAsync(RecognitionConfig config, RecognitionAudio audio, CancellationToken cancellationToken)
+            => _client.SyncRecognizeAsync(
+                GaxPreconditions.CheckNotNull(config, nameof(config)),
+                GaxPreconditions.CheckNotNull(audio, nameof(audio)),
+                cancellationToken);
+
+        public Operation<AsyncRecognizeResponse> BeginRecognize(RecognitionConfig config, RecognitionAudio audio, CallSettings callSettings = null)
+        {
+            var operation = _client.AsyncRecognize(
+                GaxPreconditions.CheckNotNull(config, nameof(config)),
+                GaxPreconditions.CheckNotNull(audio, nameof(audio)),
+                callSettings);
+            return new Operation<AsyncRecognizeResponse>(operation, _operationsClient);
+        }
+
+        public Task<Operation<AsyncRecognizeResponse>> BeginRecognizeAsync(RecognitionConfig config, RecognitionAudio audio, CancellationToken cancellationToken)
+            => BeginRecognizeAsync(config, audio, new CallSettings { CancellationToken = cancellationToken });
+
+        public async Task<Operation<AsyncRecognizeResponse>> BeginRecognizeAsync(RecognitionConfig config, RecognitionAudio audio, CallSettings callSettings = null)
+        {
+            var operation = await _client.AsyncRecognizeAsync(
+                GaxPreconditions.CheckNotNull(config, nameof(config)),
+                GaxPreconditions.CheckNotNull(audio, nameof(audio)),
+                callSettings).ConfigureAwait(false);
+            return new Operation<AsyncRecognizeResponse>(operation, _operationsClient);
+        }
+
+        // TODO: CallSettings?
+        public Operation<AsyncRecognizeResponse> PollRecognize(string operationName) =>
+            Operation<AsyncRecognizeResponse>.PollFromName(operationName, _operationsClient);
+
+        public Task<Operation<AsyncRecognizeResponse>> PollRecognizeAsync(string operationName) =>
+            Operation<AsyncRecognizeResponse>.PollFromNameAsync(operationName, _operationsClient);
+
+        public async Task<RecognizerStream> BeginStreamingRecognize(StreamingRecognitionConfig config, CallSettings settings = null)
```

Comment:
Hmm, I guess. Because we're making a request which is naturally async, I wasn't going to include a sync version... do you think that's okay? The streaming API is fundamentally pretty asynchronous.